controllers/v1/userProjects.js

```diff
@@ -1125,5 +1126,29 @@ module.exports = class UserProjects extends Abstract {
                 }
         })
     }
+    async listUserProjects(req) {
```

Comment:
rename this controller to userProjects @borkarsaish65

Reply:
done